src/Client.php

```diff
@@ -30,6 +31,17 @@ public function __construct($uri, $options, $driverOptions)
     }
 
     /**
+     * Drop a database.
+     *
+     * @param string $databaseName
+     * @return Result
+     */
+    public function dropDatabase($databaseName)
```

Comment:
If we did that and allowed the argument to be either a string or a Database instance, would we get into trouble with `dropCollection()`? I would expect a Collection's `__toString()` method to return its full namespace, as we do in the [1.x driver](http://php.net/manual/en/mongocollection.--tostring.php). I'd rather not invite the inconsistency.